test/test_ssl.py

```diff
@@ -372,3 +373,73 @@ def test_ssl_replication(pg, bouncer, cert_dir):
     # physical rep
     connect_args["replication"] = "true"
     bouncer.psql("IDENTIFY_SYSTEM", **connect_args)
+
+
+def test_servers_no_disconnect_on_reload_with_no_tls_change(bouncer, pg, cert_dir):
+    bouncer.default_db = "pTxnPool"
+
+    with bouncer.cur() as cur:
+        assert pg.connection_count(dbname="p0") == 1
+        # change nothing and RELOAD
+        bouncer.admin("RELOAD")
```

Comment:
This should be in the next `log_contains` block (same for other tests)

Reply:
[fb9bed3](https://github.com/pgbouncer/pgbouncer/pull/1157/commits/fb9bed316a0cd7b5605ef849df2654c11034bada). 

In the other tests there are `RELOAD`s in the `log_contains`block. The second and third test have one before the `log_contains`but only as part of test setup and the assertion block also has a `RELOAD` for each of those.

Reply:
Oops. I misread my tests and was wrong. Fixed. [5fe717a](https://github.com/pgbouncer/pgbouncer/pull/1157/commits/5fe717a3b1fadac6469b0bfe6d3283960591e858)